.circleci/config.yml

```diff
@@ -55,9 +81,24 @@ jobs:
             cp /home/circleci/.aws/config /home/circleci/.aws/bb/config
 
       - run:
-          name: Lint Terraform code
-          command: make tflint-deep
+          name: test2-terraform-linting-mod-a
+          command: |
+            cd modules/delegated-admin && make tflint-deep
 
+      - run:
+          name: test2-terraform-linting-mod-b
+          command: |
+            cd modules/multiaccount-setup && make tflint-deep
+
+      - slack/notify:
+          event: fail
+          mentions: '@leverage-support'
+          template: basic_fail_1
```

Comment:
@exequielrafaela Just curious, are these templates defined in CircleCI's project configuration?

Reply:
@diego-ojeda-binbash OJ you can find them here -> https://github.com/CircleCI-Public/slack-orb/tree/master/src/message_templates